Interop/Indexer/build.gradle

```diff
@@ -151,7 +153,7 @@ task updatePrebuilt {
         }
 
         copy {
-            from("$buildDir/nativelibs") {
+            from("$buildDir/nativelibs/${HostManager.host.visibleName}") {
```

Comment:
To support different stubs on different hosts? technically speaking, unless same tree is shared on different hosts shalln't be an issue.

Reply:
No, to fix the `updatePrebuilt` task: C stubs have moved to this directory.